mopidy/core/playback.py

```diff
@@ -139,64 +136,59 @@ def get_time_position(self):
     """
 
     def get_volume(self):
-        """Get the volume.
-
-        Integer in range [0..100] or :class:`None` if unknown.
-
-        The volume scale is linear.
         """
-        if self.mixer:
-            return self.mixer.get_volume().get()
-        else:
-            # For testing
-            return self._volume
+        ... deprecated:: 0.20
+            Use :meth:`core.mixer.get_volume()
+            <mopidy.core.MixerController.get_volume>` instead.
+        """
+        warnings.warn(
+            'playback.get_volume() is deprecated', DeprecationWarning)
+        return self.core.mixer.get_volume()
 
     def set_volume(self, volume):
-        """Set the volume.
-
-        The volume is defined as an integer in range [0..100].
-
-        The volume scale is linear.
         """
-        if self.mixer:
-            self.mixer.set_volume(volume)
-        else:
-            # For testing
-            self._volume = volume
+        ... deprecated:: 0.20
+            Use :meth:`core.mixer.set_volume()
+            <mopidy.core.MixerController.set_volume>` instead.
+        """
+        warnings.warn(
+            'playback.set_volume() is deprecated', DeprecationWarning)
+        return self.core.mixer.set_volume(volume)
 
     volume = deprecated_property(get_volume, set_volume)
     """
     .. deprecated:: 0.20
-        Use :meth:`get_volume` and :meth:`set_volume` instead.
+        Use :meth:`core.mixer.get_volume()
+        <mopidy.core.MixerController.get_volume>` and
+        :meth:`core.mixer.set_volume()
+        <mopidy.core.MixerController.set_volume>` instead.
     """
 
     def get_mute(self):
-        """Get mute state.
-
-        :class:`True` if muted, :class:`False` otherwise.
         """
-        if self.mixer:
-            return self.mixer.get_mute().get()
-        else:
-            # For testing
-            return self._mute
-
-    def set_mute(self, value):
-        """Set mute state.
+        ... deprecated:: 0.20
+            Use :meth:`core.mixer.get_mute()
+            <mopidy.core.MixerController.get_mute>` instead.
+        """
+        warnings.warn('playback.get_mute() is deprecated', DeprecationWarning)
+        return self.core.mixer.get_mute()
 
-        :class:`True` to mute, :class:`False` to unmute.
+    def set_mute(self, mute):
         """
-        value = bool(value)
-        if self.mixer:
-            self.mixer.set_mute(value)
-        else:
-            # For testing
-            self._mute = value
+        ... deprecated:: 0.20
+            Use :meth:`core.mixer.set_mute()
+            <mopidy.core.MixerController.set_mute>` instead.
+        """
+        warnings.warn('playback.set_mute() is deprecated', DeprecationWarning)
```

Comment:
Both in Python and WS API this is referred to as `core...`. I think it's only in the JS API that "core" is replaced with "mopidy".